perfmetrics/scripts/utils/metrics_util_test.py

```diff
@@ -0,0 +1,34 @@
+"""Test for fio output logging."""
+
+import unittest
+import os
+from metrics_util import remove_old_log_files
+
+
+class TestFioLogger(unittest.TestCase):
+
+  def test_fio_log_dir_content(self):
+    if 'fio_log_test' in os.listdir('.'):
+      os.system('rm -r fio_log_test')
+    os.mkdir('fio_log_test')
+    os.mkdir('fio_log_test/log_dir')
+    for i in range(20):
+      os.system('touch fio_log_test/log_dir/{}.txt'.format(i))
+
+    remove_old_log_files('fio_log_test/log_dir')
+
+    log_dir_content = os.listdir('fio_log_test/log_dir')
+
```

Comment:
nit: remove new line here. no extra new lines in the assert logic.

Reply:
Done